Bottom: 5508b3a49c29585d856339b8dddaf585cc5d7e66
Top:    76bdecc0f73cf5b38d238756206015fe53be007f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-07 09:56:44 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index a81ecae..989bd10 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
